taxi/forms.py

```diff
@@ -0,0 +1,63 @@
+from django import forms
+from django.contrib.auth import get_user_model
+from django.contrib.auth.forms import (
+    UserCreationForm,
+    UserChangeForm
+)
+from taxi.models import (
+    Driver,
+    Car
+)
+
+CHAR_AMOUNT = 8
+
+
+class Validator:
+    def __init__(self):
+        self.cleaned_data = None
+
+    def clean_license_number(self):
+        license_number = self.cleaned_data["license_number"]
+        if len(license_number) != CHAR_AMOUNT:
+            raise forms.ValidationError(
+                f"Invalid length of license number ({CHAR_AMOUNT})"
+            )
+        elif (not license_number[:3].isalpha()
+              or not license_number[:3].isupper()):
+            raise forms.ValidationError(
+                "Invalid license number."
+                "First 3 characters should be uppercase letters"
+            )
+        elif not license_number[3:].isdigit():
+            raise forms.ValidationError(
+                "Invalid license number. Last 5 characters should be digits"
+            )
+        return license_number
+
+
+class DriverCreationForm(UserChangeForm, Validator):
```

Comment:
The `DriverCreationForm` should inherit from `UserCreationForm` instead of `UserChangeForm`. This is because `UserCreationForm` is intended for creating new user instances, while `UserChangeForm` is for updating existing ones.